elements/rh-alert/docs/10-style.md

```diff
@@ -1,115 +1,92 @@
-be used to express more urgency or better grab the attention of a user.
+There is an alternate inline alert where the border covers all sides.
 
-<uxdot-example width-adjustment="538px">
-  <img alt="Two examples of an alternate design for inline alerts"
-       src="../alert-style-inline-alt.svg">
+<uxdot-example width-adjustment="408px">
+  <img src="../alert-style-variant-inline-alt.svg" alt="Two examples of an alternate design for inline alerts">
 </uxdot-example>
 
 ### Toast
 
-The required elements of a Toast alert are a thin top bar, icon, title, close
-button, and a white container with a subtle drop shadow. Supporting text and
-buttons may or may not be included below the title to add clarity or optional
-actions.
+Same as the inline alert, at minimum, the status icon and title needs to be visible.
 
-<uxdot-example width-adjustment="538px">
-  <img alt="Two examples of a toast alert"
-       src="../alert-style-toast.svg">
+<uxdot-example width-adjustment="456px">
+  <img src="../alert-style-variant-toast.svg" 
+        alt="Two examples of a toast alert"
+        width="456px">
 </uxdot-example>
 
-## Interaction states
-
-Interaction states are visual representations used to communicate the status of
-a component or element. The close button and any linked content are the only
-interactive elements in both alert variants.
-
-### Hover
 
-<uxdot-example width-adjustment="538px">
-  <img alt="Examples showing hover state"
-       src="../alert-interaction-states-hover.svg">
-</uxdot-example>
+## Interaction states
 
-### Focus
+Refer to these documentation pages for interaction states.
 
-<uxdot-example width-adjustment="538px">
-  <img alt="Examples showing focus state"
-       src="../alert-interaction-states-focus.svg">
-</uxdot-example>
+- [Button](/elements/button/style/#interaction-states)
 
-### Active
+## Space
 
-<uxdot-example width-adjustment="538px">
-  <img alt="Examples showing active state"
-       src="../alert-interaction-states-active.svg">
+<uxdot-example width-adjustment="456px">
+  <img src="../alert-style-spacing-1.svg" 
+      alt="Diagram of spacing for inline alerts"
+      width="456px">
 </uxdot-example>
 
-## Spacing
-
-### Inline
-
-<uxdot-example width-adjustment="538px">
-  <img alt="Diagram of spacing for inline alerts"
-       src="../alert-spacing-inline.svg">
+<uxdot-example variant="full" no-border>
+  <img src="../alert-style-spacing-2.svg" 
+      alt="Diagram of spacing for toast alerts"
+      width="1140px"">
 </uxdot-example>
 
-### Toast
-
-<uxdot-example width-adjustment="538px">
-  <img alt="Diagram of spacing for toast alerts"
-       src="../alert-spacing-toast.svg">
+<uxdot-example width-adjustment="752px"
+               alignment="left"
+               variant="full"
+               no-border>
+  <img src="../alert-style-spacing-3.svg" 
+        alt="Diagram of spacing for alerts on mobile"
+        width="752px">
 </uxdot-example>
 
 <rh-table>
```

Comment:
I don't see this table in the Figma file, but I think it's ok to keep since other doc pages have it.